php/WP_CLI/Runner.php

```diff
@@ -258,16 +259,20 @@ public function find_command_to_run( $args ) {
 				if ( count( $cmd_path ) > 1 ) {
 					$child = array_pop( $cmd_path );
 					$parent_name = implode( ' ', $cmd_path );
+					$suggestion = $this->get_subcommand_suggestion( $child, $command );
 					return sprintf(
-						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.",
+						"'%s' is not a registered subcommand of '%s'. See 'wp help %s'.%s",
 						$child,
 						$parent_name,
-						$parent_name
+						$parent_name,
+						! empty( $suggestion ) ? "\n\nDid you mean '{$suggestion}'?" : ''
```

Comment:
I had looked at what the other code is doing and didn't find any reference to `PHP_EOL`. But I agree it is preferable.

The line breaks were to match what git is doing, but I have no strong feelings about that.

So, right now, it looks like this:
```
Error: 'clu' is not a registered wp command. See 'wp help'.

Did you mean 'cli'?
```